bookwyrm_frontend/src/views/BookDetail.vue

```diff
@@ -70,7 +70,7 @@
           <!-- Avg Rating -->
           <RatingComponent
             v-bind:displayOnly="true"
-            v-bind:initialRatings="ratingsData"
+            v-bind:ratings="bookDetails.avg"
```

Comment:
I believe it should be bookDetails.avgRate, though if it works for your manual tests and your branch is caught up to main, then disregard this.

Reply:
It shouldn't be it uses the accessor not the variable and the accessor is called getavg()

Reply:
I have tested it manually, I actually always do manual tests.

Reply:
We'll have to fix that at some point. Just be sure to check for console errors as well.